src/logger/MeetingLog.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default class MeetingLog {
+  sequenceNumber: number;
+  message: string;
+  timestampMs: number;
+  logLevel: string;
+
+  constructor(sequenceNumber: number, message: string, timestampMs: number, logLevel: string) {
```

Comment:
- If you look at other "model" classes in SDK, many of them do not have a constructor unless you need a particular logic. e.g., [AudioVideoControllerState](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/audiovideocontroller/AudioVideoControllerState.ts) and [ConnectionHealthPolicyConfiguration](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/connectionhealthpolicy/ConnectionHealthPolicyConfiguration.ts).

- Also, I'm not sure this class is needed since only the `MeetingSessionPOSTLogger` is using this model internally. We do not need to expose unnecessary APIs for builders. (Note that all classes, types, and interfaces in individual files are exposed.)